code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -24,6 +25,33 @@
 	icon_state = "gag"
 	w_class = 1
 
+/obj/item/clothing/mask/muzzle/tapegag
+	name = "tape gag"
+	desc = "MHPMHHH!"
+	icon_state = "tapegag"
+	item_state = null
+	w_class = 1
+	resist_time = 150
+	species_fit = list("Unathi", "Tajaran", "Vulpkanin")
+	sprite_sheets = list(
+		"Unathi" = 'icons/mob/species/unathi/mask.dmi',
+		"Tajaran" = 'icons/mob/species/tajaran/mask.dmi',
+		"Vulpkanin" = 'icons/mob/species/vulpkanin/mask.dmi'
+		)
+
+/obj/item/clothing/mask/muzzle/tapegag/dropped(mob/living/carbon/human/user)
+	var/atom/movable/R = new /obj/item/trash/tapetrash
+	if(user.species.bodyflags & HAS_FUR)
+		R.desc += " Is that...fur?"
+	var/turf/T = get_turf(src)
+	R.loc = T
+	transfer_fingerprints_to(R)
+	playsound(src,'sound/items/poster_ripped.ogg',40,1)
+	spawn(0)

```

Comment:
may want to leave a comment why this is here, otherwise, inevitably, someone is going to say "why the hell is there a random spawn here", remove, it, and run into the problems you originally had.